flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/generated/CompileUtils.java

```diff
@@ -87,4 +88,13 @@
 			throw new RuntimeException("Can not load class " + name, e);
 		}
 	}
+
+	private static String addLineNumber(String code) {
```

Comment:
this function is just called by line 80, is this a function for debugging?

Reply:
To output more information when an error occurs.
Generally, when cook fails, it shows which line is wrong. This line number starts at 1.

Reply:
So the index of following forloop should begin from 1?

Reply:
Yes